models/configuration_and_artifacts/column_monitors_config.sql

```diff
@@ -0,0 +1,10 @@
+{{
+  config(
+    materialized = 'table',
+    transient=false,
+    unique_key = 'full_name',
+    post_hook = "{% if execute %}{{ elementary_data_reliability.upload_columns_configuration() }}{% endif %}"
+  )
+}}
+
+{{ empty_table([('full_name', 'string'), ('database_name', 'string'), ('schema_name', 'string'), ('table_name', 'string'), ('column_name', 'string'), ('column_monitors', 'string')]) }}
```

Comment:
In the rest of the project I use:
'full_table_name'
'full_column_name'
'full_schema_name'

The config is not 100% part of the project, but maybe we should be consistent anyway.